library.sh

```diff
@@ -918,7 +918,7 @@ function run_kntest() {
 # Run go-licenses to check for forbidden licenses.
 function check_licenses() {
   # Check that we don't have any forbidden licenses.
-  go_run github.com/google/go-licenses@v1.6.0 \
+  go_run github.com/google/go-licenses@v1.0.0 \
```

Comment:
We are going back?

Reply:
There are warning in various CI logs, but I haven't found `exit status 1`.